libtsuba/src/RDG.cpp

```diff
@@ -458,8 +619,12 @@ tsuba::RDG::DoMake(
       local_to_user_id_ == nullptr ? 0 : local_to_user_id_->length(),
       local_to_global_id_ == nullptr ? 0 : local_to_global_id_->length());
 
-  KATANA_CHECKED(core_->EnsureNodeTypesLoaded(rdg_dir_));
-  KATANA_CHECKED(core_->EnsureEdgeTypesLoaded(rdg_dir_));
+  if (!core_->part_header().IsEntityTypeIDsOutsideProperties()) {
```

Comment:
`Ensure{Node,Edge}TypesLoaded` are still required here; they are about loading the types of unloaded properties. an unfortunate overloading of our new language about node and edge types 😅 .

Reply:
Can we please rename them to `EnsureTypesOf{Node,Edge}PropertiesLoaded`?